pyfar/classes/audio.py

```diff
@@ -591,12 +618,19 @@ def __init__(
 
         # initialize signal specific parameters
         self._sampling_rate = sampling_rate
+
+        assert (isinstance(complex, bool))
+        self._complex = complex
         self._VALID_FFT_NORMS = [
             "none", "unitary", "amplitude", "rms", "power", "psd"]
 
         # check fft norm
         if fft_norm in self._VALID_FFT_NORMS:
-            self._fft_norm = fft_norm
+            if self._complex and fft_norm == "rms":
+                raise ValueError(("'rms' normalization is not valid for "
+                                  "complex time signals"))
```

Comment:
Why is rms not allowed for complex signals?

Reply:
Because complex time signals always have double-sided spectra, for which RMS spectra do not exist:
see Jenses paper p.4. :"Note that RMS-spectra are inherently single-sided and have no equivalent double-sided representation." I hope I have understood that correctly.

Reply:
Oh, yes. Thanks for pointing this out. I noted that we need to update https://pyfar.readthedocs.io/en/stable/concepts/pyfar.fft.html#fft-normalizations for complex signals as well. I'll make a separate issue for that.

Reply:
For this to work equally to real-valued time signals, one would need to consider the absolute value of the time data. Parseval's theorem is still valid, the rms of a complex signal is complex though.